platform/iphone/os_iphone.cpp

```diff
@@ -445,9 +449,13 @@ bool OSIPhone::can_draw() const {
 
 int OSIPhone::set_base_framebuffer(int p_fb) {
 
-	if (rasterizer_gles22) {
-		rasterizer_gles22->set_base_framebuffer(p_fb);
+	if (visual_server) {
+		visual_server->update_system_fbo(p_fb);
 	};
+
+//	if (rasterizer_gles22) {
```

Comment:
I've moved that all together, its code thats part of the old gles2 renderer, the new code is the call to update_system_fbo